examples/kubernetes.yml

```diff
@@ -59,6 +104,7 @@ spec:
   ports:
     - port: 19132
       protocol: UDP
+  type: LoadBalancer
```

Comment:
Perhaps revert this line since I provide the commented-out template at line 110. I wanted to let users default to `NodePort` type and explicitly pick `LoadBalancer`.

Reply:
Yep, nice catch. Sorry I let that slide past. The commented line is below so just let both be in there from the manual merge I had to do.